src/modules/ROOT/pages/rfc/envoy/layer-graph.adoc

```diff
@@ -0,0 +1,61 @@
+= Graph layer
+
+The graph layer helps us determine the *processing order*, keeping in mind the *dependency constraints* put in place by the resource nodes.
+
+== Graph nodes
+
+A graph node is any structure that conforms to the `envoy.Node` interface:
+
+[source,go]
+----
+Node interface {
+  // Matches checks if the node matches the given resources and **any** of the identifiers.
+  Matches(resource string, identifiers ...string) bool
```

Comment:
Currently, all implementation follow the same pattern when implementing this fn.
```
resource == n.Resource() && n.Identifiers().HasAny(identifiers...)
```

My suggestion is that we move this fn to sep. interface and call it only if it is defined. Otherwise fallback to logic in the sample above.